core/services/helper/main.py

```diff
@@ -77,11 +86,32 @@ def scan_ports() -> List[ServiceInfo]:
         return list(valid_services)
 
 
-if __name__ == "__main__":
+FASTAPI = FastAPI(
+    title="Helper API",
+    description="Everybody's helper to find web services that are running in companion.",
+    default_response_class=PrettyJSONResponse,
+)
 
-    @bottle.route("/")
-    def home() -> str:
-        services = Helper.scan_ports()
-        return r"<br>".join([service.as_html() for service in services])
 
-    bottle.run(host="0.0.0.0", port=PORT)
+@FASTAPI.get(
+    "/web_services",
+    response_model=List[ServiceInfo],
+    summary="Retrieve web services found.",
+)
+@version(1, 0)
+def web_services() -> Any:
+    """REST API endpoint to retrieve web services running."""
+    return Helper.scan_ports()
+
+
+APP = VersionedFastAPI(
```

Comment:
I would say that shadowing it here as "app" does no harm, but "versioned_app" or something like that would also be good.

Reply:
Done.